genesyscloud/journey_views/data_source_genesyscloud_journey_view.go

```diff
@@ -0,0 +1,83 @@
+package journey_views
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"terraform-provider-genesyscloud/genesyscloud/provider"
+	"terraform-provider-genesyscloud/genesyscloud/util"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/retry"
+
+	rc "terraform-provider-genesyscloud/genesyscloud/resource_cache"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+var (
+	dataSourceJourneyViewCache *rc.DataSourceCache
```

Comment:
You probably don't need a cache for the data source. We only use these for a few extremely high use data sources. @carnellj @charliecon @HemanthDogiparthi12 thoughts?

Reply:
got it, I removed the cache from the data source